src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -98,7 +98,7 @@ public enum MethodID
             "MapTo",                    // kMapTo
             "NormalizeDepth",           // kNormalizeDepth
             "Print",                    // kPrint
-            "Rank",                     // kRank
+            "__Rank",                     // kRank
```

Comment:
whats up with this change?

Reply:
It's to move Rank from being an orphan node to a List node.